packages/stateful/voting-module-adapter/react/hooks/useCw721GovernanceTokenInfoResponseIfExists.ts

```diff
@@ -0,0 +1,27 @@
+import { UseGovernanceTokenInfoResponse } from '@dao-dao/types'
+
+import { CwdVotingCw721StakedAdapter } from '../../adapters/CwdVotingCw721Staked'
+import { useVotingModuleAdapterContextIfAvailable } from '../context'
+
+// Returns the useGovernanceTokenInfo hook response if using the cwd-voting-cw721-staked
+// voting module adapter and within a voting module context. This will not error
+// if the adapter is unavailable.
+export const useCw721GovernanceTokenInfoResponseIfExists = ():
```

Comment:
is this needed? i don't see it used anywhere

Reply:
purged. was no longer being used because the gov token was already called where it was being used